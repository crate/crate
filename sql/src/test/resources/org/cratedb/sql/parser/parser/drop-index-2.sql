DROP INDEX schema1.table1.bad_index