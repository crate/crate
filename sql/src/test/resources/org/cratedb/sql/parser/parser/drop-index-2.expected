org.cratedb.sql.parser.parser.DropIndexNode@2b313906
name: schema1.table1
statementType: DROP INDEX
indexName: bad_index
existenceCheck: NO_CONDITION